uaclient/cli.py

```diff
@@ -1623,6 +1628,15 @@ def _action_refresh_contract(_args, cfg: config.UAConfig):
     print(messages.REFRESH_CONTRACT_SUCCESS)
 
 
+def _action_refresh_motd(_args, cfg: config.UAConfig):
+    # Not performing any exception handling here since both of these
+    # functions should raise UserFacingError exceptions, which are
+    # covered by the main_error_handler decorator
+    update_apt_and_motd_messages(cfg)
+    refresh_motd()
+    print(messages.REFRESH_MOTD_SUCCESS)
```

Comment:
I am still unsure if we should have any type of exception handling here

Reply:
Maybe, the only time I have run to an exception is when I compiled the package in a different distro i.e. jammy and ran it in focal, where glibc version was not correct.

Reply:
I don't think we need to worry about jammy packages erroring on focal, that makes sense I think.

@lucasmoura did you mean to modify `update_apt_and_motd_messages` to raise a UserFacingError with the `REFRESH_MOTD_FAILURE` message (a test below mocks that, but I don't see it happening in the actual code)?

We could also catch anything here and then raise a `UserFacingError(REFRESH_MOTD_FAILURE)` and log the exception itself to the log file.

Reply:
Yes, I was originally intending to raise that exception on `update_apt_and_motd_messages`, but I abandoned that idea. But I think your idea is a good one, to catch any exception and raise an specific one